templates/ovndbcluster/bin/cleanup.sh

```diff
@@ -44,3 +44,11 @@ if [[ "$(hostname)" != "{{ .SERVICE_NAME }}-0" ]]; then
     # now that we left, the database file is no longer valid
     cleanup_db_file
 fi
+
+# Stop the DB server gracefully; this will also end the pod running script.
+# By executing the stop command in the backgroung, we guarantee to exit
+# this script and not fail with "FailedPreStopHook", while the database is
```

Comment:
Do you mean that the PreStop script times out waiting for the `ovsdb-server` process to exit? Do you know why this is happening?

Reply:
No, the problem is that when we stop the ovsdb server, the init scrip ends and the pod exits. That happens before we leave this script and this is why we have this "FailedPreStopHook" error. This is a cosmetic change, because that happens at the end of this script and we actually achieve what we want, that is to stop the ovsdb server.

Reply:
Ok, with the [latest PS](https://github.com/openstack-k8s-operators/ovn-operator/pull/320/commits/ee2ec81a6d408e416e066b77ae0a00714dc56c5a), that includes all DB replicas, the kuttl tests are passing.